CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+Possible log types are `[add]`, `[bug]`, `[change]` and `[info]`.
```

Comment:
Change is something like a dependency upgrade that doesn't affect the public facing API. This could also be an info. Or the removal of a deprecated function. That's neither a feature nor a fix.

Breaking is something that could be used in addition to the other categories, e.g. `[fix/breaking]`. But in the end, I'm not sure it's needed. The version number already tells you whether there were breaking changes, and then you should look through the changelog anyways.